client/rpcserver/handlers.go

```diff
@@ -745,6 +762,21 @@ var helpMsgs = map[string]helpMsg{
     seed (string): Optional. hex-encoded 512-bit restoration seed.`,
 		returns: `Returns:
     string: The message "` + initializedStr + `"`,
+	},
+	deleteArchivedRecordsRoute: {
+		argsShort: `("unix time milli") ("matches csv path") ("orders csv path")`,
+		cmdSummary: `Delete archived matches from the database. Optionally set a time to
+    delete records after and file paths to save deleted records as comma separated
+    values. Note that file locations are from the perspective of dexc and not the caller.`,
+		argsLong: `Args:
+    unix time milli (int): Optional. If set deletes records after the date in unix time
```

Comment:
after ->  before, right?  As in older than this time.

Reply:
Yes before....